modules/rostests/apitests/atl/CAtlList.cpp

```diff
@@ -113,9 +113,54 @@ test_SwapElements()
     ok_list(list, "3,1,2,");
 }
 
+static void
+test_AppendListToTail()
+{
+    CAtlList<int> list;
+    list.AddTail(1);
+    list.AddTail(2);
+    list.AddTail(0);
+    ok_list(list, "1,2,0,");
+    
+    CAtlList<int> list_tail;
+    list_tail.AddTail(8);
+    list_tail.AddTail(1);
+    list_tail.AddTail(0);
+    ok_list(list_tail, "8,1,0,");
+    
+    list.AddTailList(&list_tail);
+    ok_list(list, "1,2,0,8,1,0,");
+    
```

Comment:
@SigmaTel71 Please remove any trailing whitespace in this file. (This line has some, for example.)